src/osd/PG.cc

```diff
@@ -5205,6 +5205,12 @@ bool PG::write_blocked_by_scrub(const hobject_t& soid)
   return true;
 }
 
+bool PG::range_intersects_scrub(const hobject_t &start, const hobject_t& end)
+{
+  return (start <= scrubber.end &&
```

Comment:
I think it should be start < scrubber.end because scrubber.end is NOT part of the range?

Reply:
yes--updated!